src/span.rs

```diff
@@ -40,18 +40,60 @@
 /// let span = span!(style; "test {:04}", 123);
 /// ```
 ///
+/// # Note
+///
+/// The first parameter must be a formatting specifier followed by a comma OR a [`Style`] followed by a semicolon.
+///
+/// For example, the following will fail to compile:
```

Comment:
It's worth noting into style here, but probably not in the error message.

Reply:
Resolved in https://github.com/ratatui-org/ratatui-macros/pull/43/commits/3402dc242518eb8127a48c283d57aa14cead6596